### ISSUE TITLE
Attempting to remove recently played game from shortcuts crashes

### ISSUE THREAD

**Issue by john-cabaj** [rileytestut/Delta#302]
When attempting to remove a game from the recently played games shortcuts, the all crashes. It’s very reproducible. 

iPhone 14 Pro. Delta 1.5.1

**Comment by imticho** [comment]
Yeah, i tried it out and it happens to me too, i think i know where its at so I'l try to take a crack at it

**Comment by john-cabaj** [comment]
It probably makes sense to not even be able to modify the recently played games list. Keep the top n and that's what you've played recently. If you elect not to use recently played games, then the last can add and remove as you see fit.

**Comment by imticho** [comment]
@rileytestut , what's the functionality of this line of code?
<img width="841" alt="Screenshot 2024-04-26 at 1 56 37 AM" src="https://github.com/rileytestut/Delta/assets/121001422/703e5d46-13b1-498e-94b1-aa6066ca6aef">
It seems that this specific line is causing the crash when removing a shortcut, from what I understand, it treats this update almost like an insertion but doesn't actually treat it as an insertion, causing a NSInternalInconsistencyException. Removing the line seems to make the removing and adding shortcuts feature work fine, but I'm not sure if there are any other dependencies in this line.
